### PR TITLE
Add ruby next-gen snippets (rest/ top -> bottom)

### PR DESCRIPTION
**Pull request by mcelicalderon**
This PR will create ruby next-gen snippets inside the `rest/` directory starting from the top.
### TODO
- [x] access-tokens
- [x] accounts
- [x] addresses
- [x] applications
- [x] authorized-connect-apps
- [x] available-phone-numbers
- [x] call
- [x] call-feedback
- [x] change-call-state
- [x] conference
- [x] connect-apps
- [x] incoming-phone-numbers
- [x] keys
- [x] making-calls
- [x] making-calls-sip
- [x] media
- [x] member
- [x] message
- [x] messages
- [x] notification
- [x] outgoing-caller-ids
- [x] participant
- [x] queue
- [x] recording
- [x] response: _This resource if of type output_.
- [x] sending-messages
- [x] short-codes
- [x] sip
- [x] sip-in
- [x] sms
- [x] subaccounts
- [x] test-credentials
- [x] token
- [x] transcription
- [x] usage-records
- [x] usage-triggers
- [x] voice

### CODE REVIEW

**Comment by mcelicalderon** [comment]
@joliveros Here are the first snippets we created. You can continue working on these snippets from top to bottom in the `rest` directory

File: rest/accounts/instance-post-example-1/instance-post-example-1.5.x.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+# Get an object from its sid. If you do not have a sid,
+# check out the list resource examples on this page
+@account = @client.api.v2010
+                  .accounts('ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
+                  .fetch
+@account.update({status: 'suspended'})
```
Comment (mcelicalderon) [inline rest/accounts/instance-post-example-1/instance-post-example-1.5.x.rb]: Redundant braces here, no need for them when the only argument is a hash

File: rest/accounts/instance-post-example-2/instance-post-example-2.5.x.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+@client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# Get an object from its sid. If you do not have a sid,
+# check out the list resource examples on this page
+@account = @client.api.v2010.accounts(account_sid).fetch
+
+@account.update({status: 'active'})
```
Comment (mcelicalderon) [inline rest/accounts/instance-post-example-2/instance-post-example-2.5.x.rb]: same here

File: rest/accounts/instance-post-example-3/instance-post-example-3.5.x.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+@client = Twilio::REST::Client.new(account_sid, auth_token)
+
+# Get an object from its sid. If you do not have a sid,
+# check out the list resource examples on this page
+@account = @client.api.v2010.accounts(account_sid).fetch
+
+@account.update({status: 'closed'})
```
Comment (mcelicalderon) [inline rest/accounts/instance-post-example-3/instance-post-example-3.5.x.rb]: same here

File: rest/accounts/list-get-example-1/list-get-example-1.5.x.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# Get your Account Sid and Auth Token from twilio.com/user/account
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+auth_token = 'your_auth_token'
+
+# Initialize Twilio client
+@client = Twilio::REST::Client.new(account_sid, auth_token)
+
+
+# Loop over accounts and print out a property for each one
+@client.accounts.list.each do |account|:
```
Comment (mcelicalderon) [inline rest/accounts/list-get-example-1/list-get-example-1.5.x.rb]: +1 on using the list resource directly. For the instance resources we will have to do massive replacements once the library is fixed

**Comment by joliveros** [comment]
Hi @juancarlospaco,
Would you mind reviewing this huge PR? You could get some help probably from @hortega

Any and all comments are welcome.

Cheers

File: .travis.yml [hunk 5]
```diff
@@ -40,12 +40,13 @@ script:
...
-    - GRADLE_OPTS="-Dorg.gradle.daemon=true"
-    - BASH=/bin/bash
-    - BASH_SOURCE=/usr/local/bin/virtualenvwrapper.sh
+  - NODE_PATH=/usr/local/lib/node_modules
+  - NODE_TLS_REJECT_UNAUTHORIZED=0
+  - FAKE_CERT_PATH=~/build/TwilioDevEd/api-snippets/twilio-api-faker/keystore/twilio_fake.pem
+  - GRADLE_OPTS="-Dorg.gradle.daemon=true"
+  - BASH=/bin/bash
+  - BASH_SOURCE=/usr/local/bin/virtualenvwrapper.sh
+  - PARALLEL=4
```
Comment (juancarlospaco) [inline .travis.yml]: Lets maintain the proper indentation on the `*.yml`, to make it more readable and consistent.

**Comment by juancarlospaco** [comment]
:+1:

File: rest/call/list-get-example-4/meta.json [hunk 6]
```diff
@@ -1,5 +1,5 @@
 {
   "title": "Retrieve Completed Calls in Period Example",
   "type": "server",
-  "testable": false
+  "testable": true
```
Comment (mcelicalderon) [inline rest/call/list-get-example-4/meta.json]: @acamino Why was this marked as not testable before?

**Comment by mcelicalderon** [comment]
@acamino Overall this looks good, just be careful with meta.json files, not sure why some of them are explicitly marked as testable: true. Also, I see a lot (if not all) of the taskrouter snippets in this PR, that might be too much to merge at once due to the limited api requests we have with github and the CMS webhook. I would probably merge this PR in two parts and monitor the number of requests left. I can share a a way to check the remaining requests if you want.

**Pull request closed without merging**